### DISCUSSION TITLE
Community in Sidebar?

### DISCUSSION THREAD

**Discussion by sarah11918** [withastro/docs#585] [Ideas]
Just throwing this out there for discussion, since I saw it on [11ty's doc site](https://www.11ty.dev/docs/) when I was running audits. What do we think of something like this?

![Screenshot 2022-05-05 12 27 53](https://user-images.githubusercontent.com/5098874/167640146-ad6b9efd-e3b0-49f5-8de0-bafa9643acc0.png)

**Comment by yanthomasdev** [comment]
Hi, sidebar expert here.

I love the idea of a community section, maybe we can even change the placement of some other stuff? We just have to be sure of what we would want to be linked in this section.

I have been working on this in a local branch, what do you guys think if the placement was something like this?
![image](https://user-images.githubusercontent.com/61414485/168449098-6b8bcf83-853e-4c07-a6d5-a2012991c1a6.png)

**Comment by delucis** [comment]
Looks fantastic @Yan-Thomas! I wonder if having **More** above **Community** would make sense as that keeps all the page-specific stuff together?

**Comment by yanthomasdev** [comment]
> I wonder if having More above Community would make sense as that keeps all the page-specific stuff together?


Hi @delucis! It would bug me that "More" is not the final section, like in a sentence, but maybe that's just me? Since "More" in my print only has "Edit/translate this page", maybe we should rename it to "Contribute"?

**Comment by delucis** [comment]
Yeah, in this scenario "Contribute" is fantastic here.

**Comment by sarah11918** [comment]
Just checking in on this one, especially now that we're looking at moving our (current) "More" stuff out of the mobile TOC placement.

So, I like these, and I think they'll absolutely be of benefit to the sidebar. Which of these Community/Contribute things do we think we SHOULD have on mobile (at all, anywhere)? Which of these do we think people will realistically (and helpfully) need pointing to if they're visiting the site on a mobile phone (in particular) and I suppose we should also consider tablet?

**Reply by delucis** [reply]
Don’t see a reason to exclude these if these are in the main menu which is behind the hamburger toggle in any case.

**Reply by sarah11918** [reply]
Then, is this at a point where a PR makes sense, @Yan-Thomas ? I think highlighting these links is great, and we could benefit from them immediately.

**Reply by yanthomasdev** [reply]
Yeah, I think we are good to PR it. I will just have to update my branch first! The only thing I would like to change is to maybe move "Astro Docs on GitHub" back to "More" (but now renamed as "Contribute"), I think opening GitHub on mobile isn't something people would normally do if not to answer comments?. I like the idea of adding "Community" as part of the hamburger menu (maybe "Contribute" as well?), but I think it's better to leave it out of the PR for now, and then we can discuss and add this after as a new PR or just as an addition to this one.

**Reply by delucis** [reply]
I would vote for having only Edit/Translate in the More menu personally as those are page-specific? Also, we could maybe switch format at that point so it’s more like this.

```
end of last paragraph of page content.

✏️ Edit this page | 🔣 Translate this page
```

(No need to PR that, but it’s an option.)

**Reply by yanthomasdev** [reply]
I added the PR: https://github.com/withastro/docs/pull/635

**Comment by sarah11918** [comment]
WE DID IT! 🥳 

Thanks, @Yan-Thomas 💜

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

I'm closing this as this seems to be resolved! Well, Community is no longer in the sidebar but in the footer... 😄
